console/console-init/ui/src/modules/msg-and-iot/dailogs/utils/utils.ts

```diff
@@ -1,11 +1,13 @@
-import { IMessagingProjectInput } from "../components/MessagingConfigurationStep";
-import { IIoTProjectInput } from "../components";
-
 /*
  * Copyright 2020, EnMasse authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
 
+import {
+  IMessagingProjectInput,
+  IIoTProjectInput
+} from "modules/msg-and-iot/dailogs";
```

Comment:
these components and interfaces should be out of the dialogs folder;

Reply:
fixed